docs/configuration.adoc

```diff
@@ -108,11 +108,7 @@ WARNING: At this moment, this strategy does not work with Java 9.
 
 `Failed` strategy just gets all tests that failed from previous executions and mark them as *important* tests to run first (_ordering_) or not filtered (_selecting_).
 
-This strategy uses the _JUnit_ XML _de-facto_ https://github.com/apache/maven-surefire/blob/master/maven-surefire-plugin/src/site/resources/xsd/surefire-test-report.xsd[report] for reading past executions.
+This strategy uses the _JUnit_ XML _de-facto_ https://github.com/apache/maven-surefire/blob/master/maven-surefire-plugin/src/site/resources/xsd/surefire-test-report.xsd[report] for reading past executions. All reports are automatically copied by the maven extension to a temp directory `${project.directory}/.reports` and when the build is finished the directory is removed.
```

Comment:
Let's make it `.smart-testing/execution/reports`

Reply:
yeah, my memory is not good :-(

Reply:
btw. I'd like to create a separated issue/PR for this... to add also the file containing changes there.
I'll do it today morning